acceptance/api/v1/service_portforward_test.go

```diff
@@ -28,37 +27,37 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-var _ = Describe("ServicePortForward Endpoint", Label("ServicePortForward"), func() {
-	var epinioHelper epinio.Epinio
+var _ = FDescribe("ServicePortForward Endpoint", LService, func() {
```

Comment:
F is a leftover?

Reply:
Yes, thanks! This issue is actually related to the bug that I showed you: https://github.com/onsi/ginkgo/issues/1221

The Focused tests should fail, but it doesn't seems the case when using Labels. So yes, it was a leftover and it was not catched by the CI for this reason. :+1: